hazelcast/src/main/java/com/hazelcast/aggregation/impl/DistinctValuesAggregator.java

```diff
@@ -48,4 +52,34 @@ public void combine(Aggregator aggregator) {
     public Set<R> aggregate() {
         return values;
     }
+
+    @Override
+    public int getFactoryId() {
+        return AggregatorDataSerializerHook.F_ID;
+    }
+
+    @Override
+    public int getId() {
+        return AggregatorDataSerializerHook.DISTINCT_VALUES;
+    }
+
+    @Override
+    public void writeData(ObjectDataOutput out) throws IOException {
+        out.writeUTF(attributePath);
+        out.writeInt(values.size());
+        for (Object value : values) {
+            out.writeObject(value);
```

Comment:
@ihsandemir
As far as I understand,  These are customer objects. This values can be anytype. No need to restrict them.
This is no different than for datas on `map.put( ? , ? )`